### ISSUE TITLE
Unhandled 'error' event

### ISSUE THREAD

**Issue by raduchiriac** [groupe-sii/generator-webpack-angular#2]
Using node __v6.7.0__ and npm __3.10.3__

Doing a simple `yo webpack-angular` throws:
```
Trying to copy from a source that does not exist: /Volumes/extra/PROJECTS/node/webpack-angular/node_modules/generator-webpack-angular/templates/app/_.babelrc
    at EditionInterface.exports._copySingle (/Volumes/extra/PROJECTS/node/webpack-angular/node_modules/mem-fs-editor/lib/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/Volumes/extra/PROJECTS/node/webpack-angular/node_modules/mem-fs-editor/lib/actions/copy.js:23:17)
    at EditionInterface.module.exports [as copyTpl] (/Volumes/extra/PROJECTS/node/webpack-angular/node_modules/mem-fs-editor/lib/actions/copy-tpl.js:10:8)
    at AppGenerator.configFiles (/Volumes/extra/PROJECTS/node/webpack-angular/node_modules/generator-webpack-angular/generators/app/index.js:85:17)
    at Object.<anonymous> (/Volumes/extra/PROJECTS/node/webpack-angular/node_modules/yeoman-generator/lib/base.js:439:23)
    at /Volumes/extra/PROJECTS/node/webpack-angular/node_modules/run-async/index.js:25:25
    at /Volumes/extra/PROJECTS/node/webpack-angular/node_modules/run-async/index.js:24:19
    at /Volumes/extra/PROJECTS/node/webpack-angular/node_modules/yeoman-generator/lib/base.js:440:9
    at runCallback (timers.js:574:20)
    at tryOnImmediate (timers.js:554:5)
```

**Comment by ValentinGot** [comment]
Arf yeah, that's hell of a fail.

Seems like there is not even half of the package that had been published on NPM ... Checking for what happened.

**Comment by ValentinGot** [comment]
Fixed, you can upgrade to **0.0.4** :)
